packs/witai/actions/text_query.py

```diff
@@ -22,6 +22,7 @@ def run(self, text, application=None):
         if not access_token:
             raise Exception('Missing API key for application %s' % application)
 
+       # pylint: disable=no-member
```

Comment:
As the commit message says - sadly `wit` library initializes some of those members lazily based on the underlying C library so pylint is not aware that those members exist.

Sadly not much more we can do here (we could write a custom pylint plugin which tells pylint about all the C library members, but that's obviously a big overkill for now and not something we want to maintain).